examples/multisig/virtual.eg.ts

```diff
@@ -31,13 +32,30 @@ const { alexa, billy, carol, david } = await createDevUsers()
 /// With this state, we can hydrate from / use an existing virtual multisig.
 /// Let's see if we can hydrate the virtual multisig state from command line arguments.
 /// If we haven't specified any virtual multisig state, we deploy a new virtual multisig.
-let { state } = parse(Deno.args, { string: ["state"] })
+const args = parse(Deno.args, { string: ["state"], boolean: ["use-stash"] })
+let state = args.state
+const useStash = args["use-stash"]
+
 if (!state) {
+  const stashAccountId = useStash
+    ? await polkadotDev.Proxy.createPure({
```

Comment:
Nit, `.createPure({` on next line

Reply:
Keep missing these. Whats the rule? Every time there is a `.` should be on new line?